tasks/managed/embargo-check/embargo-check.yaml

```diff
@@ -62,19 +71,56 @@ spec:
             exit 1
         fi
 
+        set +x # There are a lot of custom fields, don't want to pollute the logs
+        CUSTOM_RH_FIELDS="$(curl -H "Authorization: Bearer $ACCESS_TOKEN" https://issues.redhat.com/rest/api/2/field)"
+        set -x
+        CVE_FIELD="$(jq -r '.[] | select(.name=="CVE ID") | .id' <<< "$CUSTOM_RH_FIELDS")"
+        COMPONENT_FIELD="$(jq -r '.[] | select(.name=="Downstream Component Name") | .id' <<< "$CUSTOM_RH_FIELDS")"
+
         RC=0
 
         NUM_ISSUES=$(jq -cr '.releaseNotes.issues.fixed | length' "${DATA_FILE}")
         for ((i = 0; i < NUM_ISSUES; i++)); do
             issue=$(jq -c --argjson i "$i" '.releaseNotes.issues.fixed[$i]' "${DATA_FILE}")
             server=$(jq -r '.source' <<< "$issue")
             API=$(jq -r '.[] | select(.servers[] | contains("'"$server"'")) | .api' <<< "$SUPPORTED_ISSUE_TRACKERS")
-            API_URL="https://$(jq -r '.source' <<< "$issue")/${API}/$(jq -r '.id' <<< "$issue")"
-            curl --fail --output /dev/null "${API_URL}"
+            API_URL="https://"
+            # Add authentication for issues.redhat.com
+            if [ "$server" = "issues.redhat.com" ] ; then
+                API_URL="-H \"Authorization: Bearer $ACCESS_TOKEN\" ${API_URL}"
+            fi
+            API_URL="${API_URL}$(jq -r '.source' <<< "$issue")/${API}/$(jq -r '.id' <<< "$issue")"
+            OUTPUT=$(curl --fail "${API_URL}")
             if [ $? -ne 0 ] ; then
-                echo "${issue} is not publicly visible. Assuming it is embargoed and stopping pipelineRun execution."
+                echo "${issue} is not visible. Assuming it is embargoed and stopping pipelineRun execution."
                 RC=1
```

Comment:
This should probably also have `continue` after this line? Because otherwise you will try to read from OUTPUT which you failed to obtain.

Reply:
Yeah, I didn't realize I was changing the behavior to exit early now. I switched to just setting RC and adding in continues. Thanks for catching that